content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -42,14 +42,18 @@ for your specific needs. The following built-in configuration profiles are curre
 1. **preview**: the preview profile contains features that are experimental. This is intended to explore new features
                 coming to Istio. Stability, security, and performance are not guaranteed - use at your own risk.
 
+1. **openshift**: used to install Istio on OpenShift platform. Note that the OpenShift is incompatible with the Istio
```

Comment:
I believe we need to add a section called `Vendor Optimized Profiles` to include profiles like this.

See https://github.com/istio/istio/blob/master/manifests/profiles/PROFILES.md. The reason is that this profiles are supported by the vendor and not necessarily the Istio community so could see one of them going away at some point and it might not be handled in the normal community deprecation?

Reply:
Do you mean to add a separate table for Vendor profile?
Is it ok to have `opnshift` description here or it also needs to be moved below the table?

Reply:
I don't think we want anything vendor specific in these docs. Maybe add a tip here, instead of the profile:
```
{{< tip >}}
Some additional vendor-specific configuration profiles are also available.
For a complete list, refer to [Vendor Optimized Profiles]({{< github_file >}}/manifests/profiles/PROFILES.md).
{{< /tip >}}
```

Reply:
So that means, nothing needs to be added other than just a `<tip>`?

Reply:
Done.